GitUtils.ps1

```diff
@@ -115,12 +115,15 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $filesModified = @()
         $filesDeleted = @()
         $filesUnmerged = @()
+        $stashCount = 0
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
             dbg 'Getting status' $sw
             $status = git -c color.status=false status --short --branch 2>$null
+            $stashCount = $null | git stash list 2>$null | measure-object | select -expand Count

```

Comment:
> If we do go the `EnableStashStatus` route, let's do that check outside the `EnableFileStatus` check. No need for the settings to be interdependent.

Status setting looks good on the output side, but we're still doing the work even if we don't need it.